### ISSUE TITLE
ios notes uses default Branch image rather than contentImageUrl

### ISSUE THREAD

**Issue by alexbainbridge** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#339]
I create an object with contentImageUrl

When this is tweeted from the share sheet, the correct image shows
When this is added an apple iOS note (a new note), the default Branch image shows (within the note)

i.e. think I have the Branch object configured OK otherwise it wouldn't show the right image on twitter, so I am thinking it is something else not at my end

This is an example image that doesn't work
https://res.cloudinary.com/uptaste/image/upload/g_auto/experiences/leyhh0exkyigjffdnzjq

(I have tried http rather than https, I have tried a smaller image)

Any thoughts as to what it could be?

**Comment by ethanneff** [comment]
Hello Alex, 

Good find. I was able to replicate the issue on my end as well. The Branch logo always appears within Apple Notes. The good news is your image should show up correctly when shared to other apps, just not Apple Notes ([example](http://i.imgur.com/bAusfGW.jpg)).

I used the following code to test (replacing the contentImageUrl with your image URL):
- https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#create-content-reference
- https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#create-deep-link

I will continue investigating why the contentImageUrl is broken for Apple Notes.

Thanks,

**Comment by alexbainbridge** [comment]
Hi Ethan
I tried an image on different hosting (to see if somehow that was related) but that didn't APPEAR to make any difference.

Apple notes is obviously getting that image from somewhere on the Branch system so conceptually overall should work, just some Branch plumbing somewhere not right.  

Alex

**Comment by ethanneff** [comment]
Thanks for the additional testing!

I also looked into our Branch iOS SDK and it is working fine there. So something is breaking on the Branch Cordova SDK layer. I will investigate a fix. In the meantime, just know your image (https://res.cloudinary.com/uptaste/image/upload/g_auto/experiences/leyhh0exkyigjffdnzjq) should work fine when shared to other apps (e.g. Facebook, Twitter, iMessage, etc).

Thanks,

**Comment by massivekarenwong** [comment]
Hi Ethan, we ran into the same issue in our app as well. Any updates on this? Thanks!

**Comment by miklos-molnar** [comment]
Hey @ethanneff, do you have any updates on this issue?

**Comment by ethanneff** [comment]
Hello hello,

I was able to reproduce the bug, but unfortunately, this issue had been backlogged. Our main focus is how deep links behave on the internet (Safari, Chrome, Facebook, Twitter, Hangouts, iMessage, etc). The preview image for all of these apps are working correctly.

We have found in our logs that very few users share links with a Cordova app in Apple Notes. This is a test case for developers, but for the general population, this bug will never be noticed. Because of this, priority for this bug will remain low. 

I thank you all for bringing this bug to my attention.
